content/dstu2/openid-connect.md

```diff
@@ -0,0 +1,88 @@
+---
+title: OpenID Connect | DSTU 2 API
+---
+# Authorization Server OpenID Connect Support #
+---------------------------------------------
+###Introduction###
+
+[OpenID Connect][OPENID] describes itself as "a simple identity layer 
+on top of the OAuth 2.0 protocol".  The specification provides a set
+of message structures, a messaging protocol, and a security framework
+to allow a system that has authenticated a user to securely convey said
+identity to another service provider (relying party).  Cerner's 
+authorization server implements support for OpenID Connect in
+conjunction with the [SMART on FHIR<sup>�</sup>][SMART] profile 
+of OAuth 2.  Client applications may use this as a source of 
+"single sign-on" for users when protecting resources outside of the EMR 
```

Comment:
I can associate single sign-on to its abbreviation.  Are there instructions for how to generate the site itself?  I'm assuming its because I'm using a markdown-style link that points to an .md page (such that the links work when viewed in GitHub), but perhaps those aren't translated to whatever extension (lack thereof?) in the final output?